.ci/docker.sh

```diff
@@ -59,12 +59,24 @@ then
     echo DOCKER_CONTAINER_ID=${DOCKER_CONTAINER_ID}
     if [ ! -z ${DOCKER_CONTAINER_ID+x} ];
     then
+        if [[ "${DOCKER_IMAGE}" =~ slc6$ ]]
+        then
+            docker exec -ti ${DOCKER_CONTAINER_ID} /bin/bash -ec 'wget https://github.com/mexanick/wiscrpcsvc/releases/download/1.0.0/wiscrpcsvc-1.0.0-1.testing.slc6.x86_64.rpm -O wiscrpc.rpm'
+        elif [[ "${DOCKER_IMAGE}" =~ cc7$ ]]
+        then
+            docker exec -ti ${DOCKER_CONTAINER_ID} /bin/bash -ec 'wget https://github.com/mexanick/wiscrpcsvc/releases/download/1.0.0/wiscrpcsvc-1.0.0-1.testing.centos7.x86_64.rpm -O wiscrpc.rpm'
+        else
+            docker exec -ti ${DOCKER_CONTAINER_ID} /bin/bash -ec 'wget https://github.com/mexanick/wiscrpcsvc/releases/download/1.0.0/wiscrpcsvc-1.0.0-1.testing.centos7.x86_64.rpm -O wiscrpc.rpm'#need to compile cc8 for the future
+        fi
         docker exec -ti ${DOCKER_CONTAINER_ID} /bin/bash -ec 'echo Testing build on docker for `cat /etc/system-release`'
         docker logs $DOCKER_CONTAINER_ID
+        docker exec -ti ${DOCKER_CONTAINER_ID} /bin/bash -ec 'source /opt/rh/python27/enable'
```

Comment:
Can you check the filesystem structure of the built RPM from this env?
(` rpm -q --qf "[%-15{=NAME} %-36{FILENAMES} %{FILEMODES:perms}\n]" -p <rpm file>`)
My suspicion/fear is that the built RPM will explicitly require `rh/python27` to be enabled, and will install into that environment
Also, this shouldn't really be necessary except in the event that the system `python` is not `python2.7`